Makefile

```diff
@@ -103,19 +103,19 @@ mod-download:
 ## --------------------------------------
 
 .PHONY: gen
-gen: mod-download proto/agent/agent.pb.go konnectivity-client/proto/client/client.pb.go mock_gen
+gen: mod-download proto/agent/agent_grpc.pb.go konnectivity-client/proto/client/client_grpc.pb.go mock_gen
```

Comment:
This is just 2 of the 4 generated pb.go. Maybe it isn't a real problem - but putting all 4 in an intermediate target like 'gen-go' seems nicer.

Reply:
Its probably technically more correct. However given that the pairs (agent and agent_grpc) and (client and client_grpc) are generated by the same step, putting in all 4 is provably the same, at least at the moment.